xml/decoders/tcs/CV100_CV157_FW86.xml

```diff
@@ -13,8 +13,15 @@
 <!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
 <!-- for more details.                                                      -->
 <!-- version 1 - for Unified Software -->
+<!-- version 1.1 - 3-Feb-17 WestAust55 -Motor Pane Restructured plus CV182 for Special BEMF and Keep-Alive added -->
+<!-- version 1.1 - 3-Feb-17 WestAust55 -CV104 added for third Private User ID purposes -->
 <variables xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd">
-  <variable CV="105" item="User Id #1">
+  <variable CV="104" item="User Id #3">
```

Comment:
I'm not sure I understand this change.   The `item` is "User Id #3" but I don't see a programming definition for that. Perhaps a typo?